README.md

```diff
@@ -304,11 +304,14 @@ crud.select('customers', {{'<=', 'age', 35}})
 You can iterate across a distributed space using the `crud.pairs` function.
 Its arguments are the same as [`crud.select`](#select) arguments,
 but negative `first` values aren't allowed.
+Also since it's possible to return only rows without metadata
+user could pass `use_tomap` flag (`false` by default) to
+iterate over flat tuples or objects.
 
 **Example:**
 
 ```lua
-for _, obj in crud.pairs('customers', {{'<=', 'age', 35}}) do
+for _, obj in crud.pairs('customers', {{'<=', 'age', 35}}, {use_tomap = true}) do
     -- do smth with the object
 end
```

Comment:
What about adding an example w/ `use_tomap = false`.

BTW, this example doesn't show what exactly is returned.

Reply:
Rewritten to

```lua
for _, obj in crud.pairs('customers', {{'<=', 'age', 35}}, {use_tomap = false}) do
    print(json.encode(obj)) -- {5, 1172, 'Jack', 35}
end

for _, obj in crud.pairs('customers', {{'<=', 'age', 35}}, {use_tomap = true}) do
    print(json.encode(obj)) -- {id = 5, name = 'Jack', bucket_id = 1172, age = 35}
end
```

Reply:
Is it `obj` in both cases?

Reply:
Well, let's use `tuple` in the first case and `object` in the second.